src/NServiceBus.Transport.SQS/InputQueuePump.cs

```diff
@@ -45,6 +45,22 @@ class InputQueuePump : IMessageReceiver
 
         public async Task Initialize(PushRuntimeSettings limitations, OnMessage onMessage, OnError onError, CancellationToken cancellationToken = default)
         {
+            try
+            {
+                inputQueueUrl = await queueCache.GetQueueUrl(ReceiveAddress, cancellationToken)
+                .ConfigureAwait(false);
+            }
+            catch (QueueDoesNotExistException ex)
+            {
+                throw new QueueDoesNotExistException(
+                        $"Queue `{ReceiveAddress}` doesn't exist. Call endpointConfiguration.EnableInstallers() to create the queues at startup, or create them manually.",
```

Comment:
If we pass the `setupInfrastructure` flag to the pump, can we improve the exception message? E.g., when `setupInfrastructure` is true, we can throw an exception with a different message.

Reply:
I have passed the flag to the pump and adjusted the message accordingly.